src/components/ButtonDropdown.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { Button } from "components/Button";
+import Icon from "@leafygreen-ui/icon";
+import Card from "@leafygreen-ui/card";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+
+interface Props {
+  disabled?: boolean;
+  setIsVisibleDropdown?: (v: boolean) => void;
+  loading?: boolean;
+  isVisibleDropdown?: boolean;
+  dropdownItems: JSX.Element[];
+  dataCyBtn?: string;
+  dataCyDropdown?: string;
+}
+
+export const ButtonDropdown: React.FC<Props> = ({
+  disabled = false,
+  loading = false,
+  setIsVisibleDropdown = function noop(v: boolean) {
+    return;
+  },
+  isVisibleDropdown = true,
+  dropdownItems,
+  dataCyBtn = "ellipsis-btn",
+  dataCyDropdown = "card-dropdown",
+}: Props) => {
+  const toggleDropdown = () => {
+    setIsVisibleDropdown(!isVisibleDropdown);
+  };
+  return (
+    <Container>
+      <Button
+        size="small"
+        dataCy={dataCyBtn}
+        disabled={disabled}
+        loading={loading}
+        onClick={toggleDropdown}
+        glyph={<Icon glyph="Ellipsis" />}
+      />
+      {isVisibleDropdown && (
+        <Dropdown data-cy={dataCyDropdown}>{dropdownItems}</Dropdown>
+      )}
+    </Container>
+  );
+};
+interface CardItemProps {
+  disabled: boolean;
```

Comment:
Can probably inline this since its only one prop

Reply:
also not necessary